doc/src/_templates/custom-class-template.rst

```diff
@@ -0,0 +1,36 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   :noindex:
+   :members:
+   :show-inheritance:
+   :inherited-members:
+   :special-members: __call__, __add__, __mul__
```

Comment:
These shouldn't be here. Either we should include all special members or none of them. I think this is a holdover from the example template.